src/citrine/informatics/descriptors.py

```diff
@@ -235,20 +250,36 @@ class FormulationDescriptor(Serializable['FormulationDescriptor'], Descriptor):
     Parameters
     ----------
     key: str
-        the key corresponding to a descriptor
+        The key for the descriptor, which must be either 'Formulation' or 'Flat Formulation'
+        to produce valid Citrine Platform assets.
+        The two allowed values can be accessed from the `FormulationKey` enum.
 
     """
 
-    typ = properties.String('type', default='Formulation', deserializable=False)
+    typ = properties.String(
+        'type', default=FormulationKey.HIERARCHICAL.value, deserializable=False
+    )
+
+    def __init__(self, key: Union[FormulationKey, str]):
+        if not isinstance(key, FormulationKey):
+            key = FormulationKey.get_enum(key)
```

Comment:
This will throw an error if `key` is not one of the two options, yes?

Reply:
Yep

Reply:
I think we are already forcing them down a path of failed assets by using other keys so that additional client-side validation is fine.

Reply:
fair enough